woostarter/patterns/about-text-button-two-images.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * Title: Text, button and two images
+ * Slug: woostarter/text-button-two-images
+ * Categories: text, about, media
+ * Keywords: about, text, story
+ * Description: A section with text, button and two images.
+ *
+
+?>
+<!-- wp:group {"metadata":{"categories":["intro"],"name":"Text, button and two images","patternName":"a8c/intro-two-column-with-image-and-content"},"align":"full","className":"alignfull is-style-default","style":{"spacing":{"margin":{"top":"0","bottom":"0"},"padding":{"top":"0","bottom":"0"}}},"layout":{"type":"constrained","justifyContent":"center"}} -->
+<div class="wp-block-group alignfull is-style-default" style="margin-top:0;margin-bottom:0;padding-top:0;padding-bottom:0"><!-- wp:spacer {"height":"var:preset|spacing|50"} -->
+<div style="height:var(--wp--preset--spacing--50)" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer -->
+
+<!-- wp:columns {"verticalAlignment":"top","align":"wide"} -->
+<div class="wp-block-columns alignwide are-vertically-aligned-top"><!-- wp:column {"verticalAlignment":"top","layout":{"type":"constrained","justifyContent":"left","contentSize":"460px"}} -->
+<div class="wp-block-column is-vertically-aligned-top"><!-- wp:heading {"fontSize":"xx-large"} -->
+<h2 class="wp-block-heading has-xx-large-font-size">About us</h2>
+<!-- /wp:heading -->
+
+<!-- wp:paragraph -->
+<p>Our essence lies in the beauty of imperfection. Inspired by the Japanese word kiri, meaning fog, the brand embraces the blurry lines, the undefined, and the asymmetrical. We celebrate the charm of the imperfect, inviting you to see the artistry in forms that are raw, organic, and unique.</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:buttons -->
+<div class="wp-block-buttons"><!-- wp:button {"className":"is-style-outline"} -->
+<div class="wp-block-button is-style-outline"><a class="wp-block-button__link wp-element-button">Learn more</a></div>
+<!-- /wp:button --></div>
+<!-- /wp:buttons --></div>
+<!-- /wp:column -->
+
+<!-- wp:column {"verticalAlignment":"top"} -->
+<div class="wp-block-column is-vertically-aligned-top"></div>
+<!-- /wp:column --></div>
+<!-- /wp:columns -->
+
+<!-- wp:spacer {"height":"var:preset|spacing|30"} -->
+<div style="height:var(--wp--preset--spacing--30)" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer -->
+
+<!-- wp:columns {"align":"wide"} -->
+<div class="wp-block-columns alignwide"><!-- wp:column {"width":"%"} -->
+<div class="wp-block-column"><!-- wp:cover {"url":"https://kiri.mystagingwebsite.com/wp-content/uploads/2025/02/pattern-clay-pots-on-a-row-scaled.jpg","id":420,"alt":"A row of rustic clay pots hanging against a woven reed wall.","dimRatio":0,"align":"wide","style":{"color":[]}} -->
```

Comment:
This `src` attribute needs to be updated to point to the themes assets directory (example: `<?php echo esc_url( get_template_directory_uri() ); ?>/assets/images/pattern-clay-pots-on-a-row-scaled.jpg`) before merging.